client/src/app/domain/definitions/meeting-settings-defaults.ts

```diff
@@ -56,6 +56,7 @@ export const meetingSettingsDefaults: { [key: string]: any } = {
     motions_amendments_multiple_paragraphs: true,
     motions_amendments_of_amendments: false,
     motions_supporters_min_amount: 0,
+    motions_block_slide_columns: 0,
```

Comment:
The default should be 1 (see backend issue https://github.com/OpenSlides/openslides-backend/issues/1545), but the help text in the settings defintions says default is 3.

Reply:
Changed default to 3 here.